inbm-lib/inbm_common_lib/utility.py

```diff
@@ -271,3 +271,63 @@ def validate_file_type(path: list[str]) -> None:
     if os.path.exists(TEMP_EXT_FOLDER):
         shutil.rmtree(TEMP_EXT_FOLDER, ignore_errors=True)
     remove_file_list(extracted_file_list)
+
+
+def get_os_version() -> str:
+    """Get os version from os release file.
+
+    @return value of the VERSION
+    """
+    if os.path.exists(OS_RELEASE_PATH):
```

Comment:
Be sure to catch `OSError` coming from this part. Could be thrown if we get a permission issue or if `OS_RELEASE_PATH` is a directory.

```python
        with open(OS_RELEASE_PATH, 'r') as version_file:
            content = version_file.read()
```

Reply:
Added try/except to catch OSError.